### PR TITLE
Added product pitch

### PR DESCRIPTION
**Pull request by digitarald**
Does that resonate, @mykmelez @brendandahl @wfwalker @TimAbraldes ?

### CODE REVIEW

**Comment by wfwalker** [comment]
looks good to me, let's start there

**Comment by marco-c** [comment]
Maybe the *What is success* paragraph could be removed and its information added to the *Goal* paragraph. The info conveyed by those two paragraphs is really similar.

**Comment by digitarald** [comment]
@marco-c good find. I leave it for now but in the future we should fine-tune the success criteria to something more measurable.

**Comment by dbialer** [comment]
I think it would be good to build product KPIs directly into this so that we have a clear understanding of what success looks like and measure from the start.  We should instrument and build measurability directly into the MVP with Github, feedback mechanisms, and perhaps analytics. These could include:
a. Page/project visits as an indicator of interest in this project and usage.  We should have a goal of X visits per month as an indicator of usage.  What is a reasonable goal here?  5K visits per month?
b. community participation - how many non-paid staff contributors join this (as one of the goals is to increase radical participation).  We should have a goal of Y contributors by Z date.  Perhaps pull requests or commits?  How would 5-10 contributors (outside paid staff) by sometime in Q1?
c. Feedback - one of the goals is feedback.  How do we collect that?  filing issues?  Can we have a feedback form?  We should have a goal of some sort of active/useful feedback.  Not sure what would be good here.
d. How do we know it is low maintenance?  We should have a goal that other teams are updating their information around X features.

If we don't achieve goals that get us on the right track, then perhaps this isn't as useful as we think?  So part of this project should not just be coding, but achieving the usage we desire.

Any thoughts on this?

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+## Why
+
+Existing web compatibilty references, like [caniuse](http://caniuse.com/) focus on cross-browser compatibilty but don't cover un-released features and lag behind on features that already shipped. Major browser vendors ([Edge](https://dev.modern.ie/platform/status/), [Chrome](https://www.chromestatus.com/features), [Webkit](https://www.webkit.org/status.html)) are publicising their own platform status with references to their bug tracking and bug owners. Similar to Platatus' goal, Chrome tracks consensus/enthusiasm from other browsers using sources that range from bugs, to IRC logs to tweets.
+
+Platatus can send an open and clear message to developers on the state of Mozilla's platform, starting from conception of a new features to release.
+
+## What is success
+
+- Developers discover new platform features early in the process, try them out and provide feedback
```
Comment (TimAbraldes) [inline README.md]: "the process" : Which process? Standardization? Implementation in Mozilla's platform? Implementation in other browser platforms? I imagine that "try them out and provide feedback" is limited to feedback on Mozilla's implementation

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+*tomayto, tomahto*, or also **platform-status**
+
+## Goal
+
+Platatus gives developers a cross-platform and cross-browser roadmap for browser features, from standardization, development to release. Not only being a channel to raise awareness, it can also drive adoption and improve the vital feedback loop Firefox's platform needs to have with web developers. For most developers it will be a reference to discover and learn, but it has the potential to provide the entry points for radical participation on Mozilla's platform development.
+
+## Why
+
+Existing web compatibilty references, like [caniuse](http://caniuse.com/) focus on cross-browser compatibilty but don't cover un-released features and lag behind on features that already shipped. Major browser vendors ([Edge](https://dev.modern.ie/platform/status/), [Chrome](https://www.chromestatus.com/features), [Webkit](https://www.webkit.org/status.html)) are publicising their own platform status with references to their bug tracking and bug owners. Similar to Platatus' goal, Chrome tracks consensus/enthusiasm from other browsers using sources that range from bugs, to IRC logs to tweets.
```
Comment (TimAbraldes) [inline README.md]: nit: no comma necessary: "that range from bugs to IRC logs to tweets."

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
-# platatus
+# Project Platatus
+
+*tomayto, tomahto*, or also **platform-status**
+
+## Goal
+
+Platatus gives developers a cross-platform and cross-browser roadmap for browser features, from standardization, development to release. Not only being a channel to raise awareness, it can also drive adoption and improve the vital feedback loop Firefox's platform needs to have with web developers. For most developers it will be a reference to discover and learn, but it has the potential to provide the entry points for radical participation on Mozilla's platform development.
```
Comment (TimAbraldes) [inline README.md]: nit: "from standardization to development to release" or "from standardization and development to release"

Comment (TimAbraldes) [inline README.md]: nit: Remove "being" in "Not only being a channel to raise awareness, [...]"

File: README.md (hunk 2 above)
Comment (TimAbraldes) [inline README.md]: nit: Either remove the comma before "like caniuse" or add a comma at the end of it

File: README.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+
+## Why
+
+Existing web compatibilty references, like [caniuse](http://caniuse.com/) focus on cross-browser compatibilty but don't cover un-released features and lag behind on features that already shipped. Major browser vendors ([Edge](https://dev.modern.ie/platform/status/), [Chrome](https://www.chromestatus.com/features), [Webkit](https://www.webkit.org/status.html)) are publicising their own platform status with references to their bug tracking and bug owners. Similar to Platatus' goal, Chrome tracks consensus/enthusiasm from other browsers using sources that range from bugs, to IRC logs to tweets.
+
+Platatus can send an open and clear message to developers on the state of Mozilla's platform, starting from conception of a new features to release.
```
Comment (TimAbraldes) [inline README.md]: nit: remove 'a' in "conception of a new features"

"from conception of new features" makes it sound like we will track new technologies that Mozilla comes up with that we want to approach a standards body about but haven't yet. Is that actually part of our goals or do we intend to start tracking once the standards body has been approached?

File: README.md [hunk 5]
```diff
@@ -1 +1,23 @@
...
+
+## What is success
+
+- Developers discover new platform features early in the process, try them out and provide feedback
+- Developers adopt bleeding edge standards because they get clear signals from browser vendors
```
Comment (TimAbraldes) [inline README.md]: From what's written above, it sounds like developers are already getting clear signals about implementation status from other browser vendors. Is the "clear signals from browser vendors" meant about Mozilla specifically? Or maybe it means that there will be clear signals of intent prior to actual implementation?

**Comment by TimAbraldes** [comment]
Looks good overall! I tried to fine-tooth-comb as much as possible for nits

**Comment by TimAbraldes** [comment]
> We should have a goal of X visits per month as an indicator of usage

Sounds good to me. I have no idea what a reasonable number would be... I also don't really know _when_ it would be reasonable to expect usage to pick up.

> We should have a goal of Y contributors by Z date. Perhaps pull requests or commits? How would 5-10 contributors (outside paid staff) by sometime in Q1?

I like the idea of making contribution a goal. These numbers sound optimistic to me but I don't know if this project will turn out to be more "if you build it they will come" or "if you start building it they will come and help out"

> Feedback - one of the goals is feedback. How do we collect that? filing issues? Can we have a feedback form? We should have a goal of some sort of active/useful feedback. Not sure what would be good here.

Good points/questions. We should probably file an issue for this and continue the discussion there.

> How do we know it is low maintenance?

This seems like a hugely important piece of the project. If it's not low-maintenace then eventually it will fizzle. Maybe a UX person can help us with this? Measure the number of clicks to update something? Measure the number of manual tasks? This also seems like a good candidate for a separate issue where we can discuss

> So part of this project should not just be coding, but achieving the usage we desire

I totally agree with this message

File: README.md (hunk 5 above)
Comment (digitarald) [inline README.md]: I feel the other browsers already share their intents well on their platform sites (some clearer than others) and Mozilla needs to do the same. Right now is is scattered across bugs, internal tools and even [private blogs](https://blog.wanderview.com/blog/2015/06/19/intent-to-implement-streams-in-firefox/).

**Pull request merged**